static/js/zamboni/devhub.js

```diff
@@ -1422,3 +1425,31 @@ function initCCLicense() {
     }));
     licenseChangeHandler();
 }
+
+function initSourceSubmitOutcomes() {
+    $('#submit-source #id_has_source input').change(function() {
+        $('#option_no_source').hide();
+        $('#option_yes_source').hide();
+        $('#submit-source #id_has_source input').each(function(index, element) {
+            var $radio = $(element);
+            if ($radio.val() == "yes" & $radio.prop('checked')) {
```

Comment:
`&&` instead of `&` here and in the other 2 conditions below (it works right now because both parts of the conditions are actual booleans, but let's not rely on that...)

Reply:
Javascript 🙄